specification.md

```diff
@@ -1634,6 +1640,70 @@ immediately after successful connection each Agent will acquire their own unique
 connection credentials. This way individual Agent's credentials may be revoked
 without disrupting the access to all other Agents.
 
+#### Agent-initiated CA trust Flow
+
+Status: [Development]
+
+This is an Agent-initiated flow that allows an agent to connect to a Server
+without the possesion of the Server's CA initially. Initially the agent will
```

Comment:
BTW, I think this same flow can also work for _replacing_ the CA. So even if the Agent knows a CA that the Server uses initially, the Server can offer a different CA to be used in the future. The difference will be that the initial connection will be with CA verification. Do we want to generalize this flow to also allow this use case?